doc/contributing/maintaining-openssl.md

```diff
@@ -157,34 +125,17 @@ files if they are changed before committing:
 
 ```console
 % git add deps/openssl/config/archs
-% git add deps/openssl/openssl/include/crypto/bn_conf.h
-% git add deps/openssl/openssl/include/crypto/dso_conf.h
-% git add deps/openssl/openssl/include/openssl/opensslconf.h
+% git add deps/openssl/openssl
 % git commit
 ```
 
 The commit message can be written as (with the openssl version set
 to the relevant value):
 
-### OpenSSL 1.1.1
-
-```text
- deps: update archs files for OpenSSL-1.1.1
-
- After an OpenSSL source update, all the config files need to be
- regenerated and committed by:
-    $ make -C deps/openssl/config
-    $ git add deps/openssl/config/archs
-    $ git add deps/openssl/openssl/include/crypto/bn_conf.h
-    $ git add deps/openssl/openssl/include/crypto/dso_conf.h
-    $ git add deps/openssl/openssl/include/openssl/opensslconf.h
-    $ git commit
-```
-
 ### OpenSSL 3.0.x
```

Comment:
Maybe it doesn't in this PR, but the heading here is not consistent with https://github.com/nodejs/node/blob/7bf50bc26aa01308d06ffd6bd75864c29486fa0b/doc/contributing/maintaining-openssl.md#L95
```suggestion
### OpenSSL 3.x.x
```